### ISSUE TITLE
Can't get it to work

### ISSUE THREAD

**Issue by digitalrep** [kamilkp/angular-sortable-view#115]
http://codepen.io/digitalrep/pen/QdQZEQ

Did I miss something?

**Comment by CBillings92** [comment]
I don't think it handles lists that span multiple columns AND rows at the same time well. If you were to comment out your `box` CSS class (so the `div`s rest on top of each other in a single, vertical stack) then it seems to work as intended (and more specifically you can juts remove the `float:left` to get it to work). 

I'm not sure why it does this, I just found this library yesterday and ran into the same issue but this would be awesome functionality to have.
